plugin/trino-fission-functions/src/main/java/io/trino/plugin/fission/FissionFucntionConfigProvider.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.fission;
+
+import java.io.IOException;
+
+public class FissionFucntionConfigProvider
+{
+    private static final String FISSION_FUCNTION_BASE_URL_ENV_VAR = "FISSION_FUCNTION_BASE_URL";
+
+    private static String fissionFunctionbaseUrl;
+
+    private FissionFucntionConfigProvider() {}
+
+    public static String getFissionFunctionBaseURL() throws IOException
+    {
+        if (fissionFunctionbaseUrl != null) {
```

Comment:
```suggestion
        if (this.fissionFunctionbaseUrl != null) {
```
Because our coding standards for CLSM that you've probably never seen say we always use "this" explicitly. :D

Reply:
Yeah... You know how hard I pushed for this on HS. But I feel like we should just abide by the best practices / guidelines of whatever project we are contributing to. Mind you, in this case, we plan to make this plugin into its own repo one day. So I'm not sure whether we should be having this conversation now or "one day"...